src/containers/Member/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Fragment } from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import MemberSidebar from 'components/MemberSidebar';
+import EditProfile from './EditProfile';
+import DailyReports from './DailyReports';
+import NewDailyReport from './NewDailyReport';
+
+const Member = () => {
+  const pathName = location.pathname;
```

Comment:
Should  be destructing this variable:
`const Member = ({ pathname }) => {`

And if you do this, you don't need to`return` keyword. 
If no logic in this component, pls using stateless component instead.
Like this: 


```
const NewDailyReport = () => (
  <ContentWrapper>
    <ContentHeader>Write a Daily Report</ContentHeader>
  </ContentWrapper>
);
```